Openldap and openldap-admin

fernandosanchezmunoz opened the pull request:
add   "packagingVersion": "3.0" on both; "minDcosReleaseVersion": "1.7" on openldap for storage
add "networking" section. Add "external_access" flag to enable VIP on Marathon-LB optionally.
add "storage" section. add "persistence" flag to optional create local persistent volumes.
change to TCP healthcheck.
fix docker versions where possible (some images just offer latest)
add icons
DCOS_PACKAGE_IS_FRAMEWORK=false
bump to version 0.3

pyronicide commented on repo/packages/O/openldap-admin/2/package.json:
```diff
@@ -0,0 +1,22 @@
+    "ldap",
+    "directory"
+    ],
+  "preInstallNotes": "Open LDAP Admin requires at least 0.1 CPU and 128MB of RAM.\n\nWARNING: OpenLDAP Admin on DCOS is currently EXPERIMENTAL. There may be bugs, incomplete features, incorrect documentation, or other discrepancies.\n\nThis package will automatically connecto to an existing DC/OS `openldap` package through an automated Virtual IP.\n\nThis package is accessible through the DC/OS ` ...
```
This will need a servicePort and no vhost if you really mean to have it accessed here.

fernandosanchezmunoz replied:
corrected -- removed vhost (openldap is TCP not HTTP).
Made external connectivity a separate section for clarity.
Made "enabled" default to false.
Added notes on unencrypted connection.

pyronicide commented on repo/packages/O/openldap/2/package.json:
```diff
@@ -0,0 +1,24 @@
+    "ldap",
+    "directory"
+  ],
+  "preInstallNotes": "OpenLDAP requires at least 0.3 CPU and 128MB of RAM.\n\nWARNING: OpenLDAP on DCOS is currently EXPERIMENTAL. There may be bugs, incomplete\nfeatures, incorrect documentation, or other discrepancies.\n\nIf you didn't provide a value for `host_volume` when installing the package,\nYOUR DATA WILL NOT BE SAVED IN ANY WAY.\n\nThis DC/OS OpenLDAP package creates a named Virtual IP ...
```
Sure you want this available via. a vhost?

fernandosanchezmunoz replied:
removed as LDAP is TCP not HTTP.

fernandosanchezmunoz commented:
added:

separate sections for "persistence" and "external_access" for clarity
visible comments on unencrypted external connections
removed VHost for postgres (which is TCP) -- not for the admin interface (HTTP)
Improved documentation
